dev-libs/log4cxx/log4cxx-0.10.0~git.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="A logging framework for C++"
+DESCRIPTION="Apache Log4cxx is a framework which provides the capability to \
+log messages in a variety of formats to the local console, local files, \
+streamed over a socket or even launch an email based on a hierarchy of \
+notification levels."
+HOMEPAGE="https://logging.apache.org/log4cxx/latest_stable/index.html"
+COPYRIGHT="2004-2007 The Apache Software Foundation"
+LICENSE="Apache v2"
+REVISION="1"
+gitSrcRevision="5f825186936a1876f92b88b371334ff26e997287"
+SOURCE_URI="https://github.com/apache/logging-log4cxx/archive/$gitSrcRevision.zip"
+CHECKSUM_SHA256="c6ab2b7d124fdf62d66d3aeda74e1559d71b3363fbe4520a134f1574b5663115"
+SOURCE_FILENAME="log4cxx-$gitSrcRevision.zip"
+SOURCE_DIR="logging-log4cxx-$gitSrcRevision"
```

Comment:
Suggestion:
```diff
-gitSrcRevision="5f825186936a1876f92b88b371334ff26e997287"
+srcGitRev="5f825186936a1876f92b88b371334ff26e997287"
-SOURCE_URI="https://github.com/apache/logging-log4cxx/archive/$gitSrcRevision.zip"
+SOURCE_URI="https://github.com/apache/logging-log4cxx/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="c6ab2b7d124fdf62d66d3aeda74e1559d71b3363fbe4520a134f1574b5663115"
+CHECKSUM_SHA256="763b07c7a487ff2c42d0e82f7c3fd598083d08dc9f5d8652990295a87d99ddb2"
-SOURCE_FILENAME="log4cxx-$gitSrcRevision.zip"
+SOURCE_FILENAME="log4cxx-$portVersion-$srcGitRev.tar.gz"
-SOURCE_DIR="logging-log4cxx-$gitSrcRevision"
+SOURCE_DIR="logging-log4cxx-$srcGitRev"
```
Advantages:
* Most recipes which use a git snapshot define `srcGitRev`.
* The `.tar.gz` is smaller than the zip archive.
* Including `$portVersion-` in `SOURCE_FILENAME` is slightly better (imo) than having only the `$srcGitRev`.

([HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering))